templates/master/01-master-container-runtime/_base/files/crio-conf-wipe.yaml

```diff
@@ -0,0 +1,4 @@
+mode: 0644
+path: "/etc/crio/crio.conf"
+contents:
+  inline: ''
```

Comment:
per @yuqi-zhang 's comments, I think we should drop this

Reply:
Only as long as we ensure that we are shipping no such file within RPM. Can we get confirmation of that somehow?

Reply:
I had attempted to drop the file from the rpm but I broke the rpm in the process, I think I've fixed. once it's trickled into the images I'll build a cluster to prove it's been done :)